packages/frontend-shared/src/gql-components/HeaderBarContent.vue

```diff
@@ -65,6 +65,7 @@
               <template
                 v-if="props.gql?.currentProject?.branch"
               >
+                {{ ' ' }}
```

Comment:
Eh, that's a little weird. I suggest adding margin-right CSS tbqh. `mr-2` or something.

Reply:
Ugh, margin causes worse overflow. Just add the space and a comment above this please 

```suggestion
                <!-- Using a margin here causes different overflow problems.
                        See PR #21325. Using a space for now. -->
                {{ ' ' }}
```

![image](https://user-images.githubusercontent.com/2801156/166838816-a4c5fec3-097b-47c7-9eb4-37d8d87612e3.png)